### ISSUE TITLE
updating the timing page in the doc + np & tf timings

### ISSUE THREAD

**Issue by edouardoyallon** [kymatio/kymatio#439]

**Comment by MuawizChaudhary** [comment]
@lostanlen Do your benchmarks take care of this automatically?

**Comment by edouardoyallon** [comment]
@janden what do we decide about this one?(keep it this way?) what should we do about the ASV branch?

**Comment by janden** [comment]
So it should just be a matter of running ASV on a few standard machines and updating the timings in the docs manually. That branch (you're talking about [this one](https://github.com/eickenberg/kymatio/tree/add_asv_to_circle_ci), right?) will eventually automate pushing the latest ASV results to a separate website, but we're pretty far off right now.

**Comment by edouardoyallon** [comment]
ok so this will be the last issue to close.

**Comment by janden** [comment]
Sorry, but this needs to be in 0.2 since the tests have changed somewhat.

**Comment by eickenberg** [comment]
does anybody know how to do this?
I can go and try to figure it out, but it might take me some hours

**Comment by janden** [comment]
I can run ASV on some of the FI nodes (V100, P100 or K40) and report the results. Thing is, we need the other baselines: ScatNet (I can do this one too), MATLAB [OM15] (what is this, ScatNet*Light*?), and PyScatWave (is this still runnable?). We might also want ASV results on some consumer card, like 1080Ti.

**Comment by eickenberg** [comment]
yeah so how do we distribute this easiest?
Is asv set up that we can just run a command on different machines?

**Comment by janden** [comment]
Log into the machine, then (assuming you've `pip install`ed `asv`)
```
cd benchmarks
asv run
```
and keep pressing enter. The results are then stored with the hostname in the folder `results`.

**Comment by eickenberg** [comment]
cool, trying that out now

**Comment by janden** [comment]
Like I said, I can run it on the cluster tomorrow and do the MATLAB timing as well. It's more the other stuff that I need help with.

**Comment by eickenberg** [comment]
> the other stuff 

I can do 1080ti

Maybe @edouardoyallon can do PyScatWave and/or ScatNetLight?

**Comment by eickenberg** [comment]
asv actually doesn't run for me. It's can't create the environment it wants to create. Because it can't find pytorch. Do you not get this error? It might be a conda thing

**Comment by janden** [comment]
I got it on my laptop too. Not sure what's going on. Could be that torch has to be installed through the torch channel (not the standard conda one).

**Comment by eickenberg** [comment]
Yep, so `conda-channels` was commented. I uncommented, which gives access to conda-forge, and it still fails. Let me see if I can add the pytorch channel there

**Comment by eickenberg** [comment]
Change the environment engine from `conda` to `virtualenv` and now it looks like it's running

**Comment by janden** [comment]
Huh. Weird. Can you make a PR?

**Comment by edouardoyallon** [comment]
Maybe we can automate the ASV thing to add some content to a webpage? I've added 2 config file, one for CUDA, one for CPU.
